Dockerfile

```diff
@@ -1,31 +1,52 @@
-FROM python:3.7-slim-stretch
+# First build the Helm binding
+FROM golang:1.12.9-stretch AS go-builder
 
 RUN mkdir /kapitan
 WORKDIR /kapitan
-COPY kapitan/ kapitan/
-COPY requirements.txt ./
+COPY . .
 
-RUN apt-get update && apt-get install -y \
-    build-essential git wget \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* \
-    && pip install --upgrade pip \
-    && pip install -r requirements.txt
+RUN ./kapitan/inputs/helm/build.sh
```

Comment:
As a good practice, you might have to `chmod +x` this script so it's always executable?